src/components/SwapForm/SwapModal/SwapBrief.tsx

```diff
@@ -26,76 +32,134 @@ const TruncatedText = styled(Text)`
   font-weight: 500;
 `
 
-const SwapBrief: React.FC<Props> = ({ inputAmount, outputAmountFromBuild, $level, isLoading, currencyOut }) => {
+export const CurrencyInputAmountWrapper = styled(Flex)`
+  flex-direction: column;
+  gap: 8px;
+  border-radius: 16px;
+  border: 1px solid ${({ theme }) => theme.border};
+  padding: 12px 16px;
+`
+
+export const ArrowDownWrapper = styled.div`
+  border: 1px solid ${({ theme }) => theme.border};
+  background: ${({ theme }) => theme.buttonGray};
+  border-radius: 50%;
+  width: 20px;
+  height: 20px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  top: calc(76px - 6px);
+  left: 50%;
+  transform: translateX(-50%);
+`
+
+export default function SwapBrief({
+  inputAmount,
+  amountInUsd,
+  outputAmount,
+  outputAmountFromBuild,
+  amountOutUsdFromBuild,
+  $level,
+  isLoading,
+  currencyOut,
+}: Props) {
   const theme = useTheme()
-  const { typedValue, feeConfig } = useSwapFormContext()
+  const { typedValue } = useSwapFormContext()
 
   const renderOutputAmount = () => {
     if (isLoading) {
       return (
         <Skeleton
-          width="120px"
+          width="108px"
           // there's border of 1px
-          height="22px"
+          height="26.5px"
```

Comment:
ugly number

Reply:
But it's worked.

Reply:
0.5px is exactly here.

Reply:
how about 26px?
26px vs 26.5px, I think there is no difference